app/models/support_interface/ucas_matches_filter.rb

```diff
@@ -36,5 +41,18 @@ def year_filter
         options: cycle_options,
       }
     end
+
+    def action_needed_filter
+      {
+        type: :checkboxes,
+        heading: 'Action needed',
+        name: 'action_needed',
+        options: [{
+          value: 'yes',
+          label: 'Yes',
+          checked: applied_filters[:action_needed]&.include?('yes'),
```

Comment:
Very minor, but you could change the URL to end with `&action_needed=anything` and have the filter apply but the box not be checked. Can we make the check for applying the filter and checking this be the same?

Reply:
Good spot, amended.